env_support/cmake/custom.cmake

```diff
@@ -43,14 +35,21 @@ endif()
 # Include root and optional parent path of LV_CONF_PATH
 target_include_directories(lvgl SYSTEM PUBLIC ${LVGL_ROOT_DIR} ${LV_CONF_DIR})
 
+# Include /examples folder
 
 if(NOT LV_CONF_BUILD_DISABLE_THORVG_INTERNAL)
     add_library(lvgl_thorvg ${THORVG_SOURCES})
     add_library(lvgl::thorvg ALIAS lvgl_thorvg)
     target_include_directories(lvgl_thorvg SYSTEM PUBLIC ${LVGL_ROOT_DIR}/src/libs/thorvg)
-    target_link_libraries(lvgl_thorvg PUBLIC lvgl)
+    if(LV_CONF_PATH)
```

Comment:
Please add back the original `target_link_libraries(lvgl_thorvg PUBLIC lvgl)`. The compile definitions added to lvgl will also take effect on lvgl_thorvg.

Reply:
Added back and tested to work